ctpolicy/chromepolicy.go

```diff
@@ -24,29 +24,21 @@ import (
 type ChromeCTPolicy struct {
 }
 
-// updateIfNotNil picks err argument iff it is not nil, oldErr otherwise.
-func updateIfNotNil(oldErr error, err error) error {
-	if err != nil {
-		return err
-	}
-	return oldErr
-}
-
 // LogsByGroup describes submission requirements for embedded SCTs according to
 // https://github.com/chromium/ct-policy/blob/master/ct_policy.md#qualifying-certificate.
-// Error warns on inability to reach minimal number of Logs requirement due to
-// inadequate number of Logs within LogList.
+// Returns an error if loglist provided is not sufficient to satisfy policy.
```

Comment:
How about `Returns an error if it's not possible to satisfy the policy with the provided loglist`?

Reply:
Updated.